devtools/create_xeen/en_constants.h

```diff
@@ -1987,4 +1988,76 @@ class EN : public LangConstants {
 			   "c\t000\v180Press a Key!\x3"
 			   "l\fd";
 	}
+
+	class EN_KeyConstants : public KeyConstants {
+	public:
+		class EN_DialogsCharInfo : public DialogsCharInfo {
+		public:
+			const int KEY_ITEM()     { return Common::KEYCODE_i; }
```

Comment:
Not sure, but I think some compilers might generate warnings over the redundant use of const for an int return. Unless you meant to do it like "int KEY_ITEM() const", which makes sense since the methods don't have any side-effects.

Reply:
const removed